### DISCUSSION TITLE
Two Way Communication?  Knob turn on screen while turning the knob on the synth?

### DISCUSSION THREAD

**Discussion by mikefix1** [RomanKubiak/ctrlr#689] [Q&A]
I'm really struggling trying to get two way communication working.  

I've made a panel and can change the values on the screen and it sends the change to the synth, but not vise versa.  I turn the knob on the synth and the screen knob doesn't change.

I've tried with a DSI Tetra and a Waldorf Pulse 2.  Neither seem to change the screen knob.  I see the MIDI messages in the MIDI monitor, just no visual change.

OS X, devices connected via USB, and devices selected in the MIDI menu correctly.

Any help would be greatly appreciated?  THANK YOU!

**Comment by dobo365** [comment]
Hi. Can you share your panel or put screenshots of the slider settings?
In the Midi monitor, you confirm that you see both incoming and outgoing messages?
Is your slide muted?
Is the received message the expected one in terms of value, CC number...
We'll find the reason, no worry ;-)
